app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -576,6 +593,95 @@ private void fetchData(boolean fromBottomSheet) {
                 });
     }
 
+    private void sortConversations(List<AbstractFlexibleItem> callItems) {
+        if (CapabilitiesUtil.hasSpreedFeatureCapability(currentUser, "last-room-activity")) {
```

Comment:
Nextcloud 14, which is the minimum already, so drop the else?

Reply:
@timkrueger as expected the deletion of this "last-room-activity" check led to the deletion of "CallItem" which in turn led to some refactoring which might affect the UserStatus PR, see commit 70b4eed6541e6d013b8104159faf01483b8b3a18
So maybe keep an eye on that while reviewing..

Reply:
> So maybe keep an eye on that while reviewing..
I'll rebase that on the UserStatus PR when tthis one is on the master. 

I'll review this tomorrow.